docs/docker-in-30-mins.md

```diff
@@ -0,0 +1,127 @@
+# Docker in 30 mins with a Node.JS application #
+
+To start using docker for your application(s), you will need Docker installed on your machine and optionally a Dockerhub account to host your docker image.
+
+
+## Installing Docker ##
+
+Install docker. A list of the supported platforms and instructions for installing docker on those platforms are located [here](https://docs.docker.com/engine/installation/#supported-platforms).  
```

Comment:
It seems a little weird that the first step of installing docker is to install docker.  Maybe the header should be "prerequisites" or something like that.